packages/ui/src/working-groups/components/Roles/MyRoleAccount.tsx

```diff
@@ -23,10 +23,12 @@ export const MyRoleAccount = ({ account }: MyRoleAccountProps) => {
             <Label>Total balance</Label>
             <TokenValue value={balance?.total} />
           </RoleAccountBalance>
-          <RoleAccountBalance>
-            <Label>Locked balance</Label>
-            <TokenValue value={balance?.locked} />
-          </RoleAccountBalance>
+          {account.name === 'Stake Account' && (
+            <RoleAccountBalance>
+              <Label>Locked balance</Label>
+              <TokenValue value={balance?.locked} />
+            </RoleAccountBalance>
+          )}
```

Comment:
This should come from the parent component. So maybe a different way to tell this component which accounts to show. AFAICS, there are 3 variants:

*   [total](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=1753%3A6533)
*   [free + total + locked](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=1753%3A6533)
*   [total + locked](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=1740%3A92222)

Besides, the `name` is just a string, so it can be anything (maybe translated in the future?)

I'd go with either

*   2 booleans: `showFree`, `showLocked`
*   one array: `showAccounts: AccountType[]` , where `AccountType = 'total' | 'locked' | 'free'`